roundtrip_test.go

```diff
@@ -38,11 +60,13 @@ func TestRoundtripStructs(t *testing.T) {
 		D: []string{"Lorem", "Ipsum"},
 		E: []Two{Two{"First"}, Two{"Second"}, Two{"Third"}},
 		F: []*Two{&Two{"Stuff"}},
-		//		G: []ConvolutedIndirectable{&Indirector{1}},
+		G: &Indirector{1},
```

Comment:
It would be good to test with two different structures which implement `ConvolutedIndirectable` over here to make sure the tags really work.